### DISCUSSION TITLE
[Feature] Warning when not using Chrome

### DISCUSSION THREAD

**Discussion by JesseWebDotCom** [immich-app/immich#5137] [Feature Request]
### The feature

If Chrome is the supported web browser, then a warning should be displayed when using a non-supported web browser. Without the warning, the user has no idea why things like videos are not playing properly.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by martabal** [comment]
What does not work without chrome ?

**Reply by JesseWebDotCom** [reply]
Some videos do not play properly in safari (preview will have the little red upper right icon and full video will not play at all - but it fine in chrome). @alextran1502 even mentioned this (ie use chrome instead) to me on Reddit.

If that's the case then it would be nice to be warned and directed to a compatible browser.

**Reply by martabal** [reply]
If this is videos, isn't it related to video codecs ? If so I think a warning in the admin section is enough. No need to scare everyone

**Reply by JesseWebDotCom** [reply]
But my concern here is for the users (not the admins who should be aware of such issues). For example, I don't want my family members going to Immich, see how no videos play, and never go back again.

Maybe you can have this as toggle in the administration section that shows the warning for all users when toggle on

**Reply by bo0tzz** [reply]
Have you changed any of the transcoding settings?

**Reply by JesseWebDotCom** [reply]
Issue is in safari (don't have Firefox). I am now fully re-encoding all videos (instead of ones not in desired format) - I'll let you know how it goes.

**Comment by aviv926** [comment]
I use Firefox and it runs Immich better than Chrome
Try disabling browser extensions and see if the problem appears

**Reply by Solarunit** [reply]
Everything works fine for me in Firefox too.

**Comment by Solarunit** [comment]
Maybe it would be better to make it work in Firefox if there is an issue?
